gen/nvml/generateapi.go

```diff
@@ -418,3 +433,76 @@ func isPublic(name string) bool {
 	}
 	return unicode.IsUpper([]rune(name)[0])
 }
+
+func generateHandleHelpers() (string, error) {
+	var builder strings.Builder
+
+	// Generate helper for each type (only if Type starts with 'nvml')
+	for _, p := range GeneratableInterfaces {
+		if !strings.HasPrefix(p.Type, "nvml") {
+			continue
+		}
+		helper, err := generateHandleHelper(p)
+		if err != nil {
+			return "", err
+		}
+		builder.WriteString(helper)
+		builder.WriteString("\n")
+	}
+
+	return builder.String(), nil
+}
+
+func generateHandleHelper(input GeneratableInterfacePoperties) (string, error) {
+	var builder strings.Builder
+
+	// Function comment
+	builder.WriteString(fmt.Sprintf("// %sHandle attempts to convert a %s to an %s.\n",
```

Comment:
This may be easier to read if we use go templates instead.

Reply:
Yes, definitely. Updated.

Reply:
I just noticed your comment is in the original generateapi.go file (which is not part of the PR). I added the template stuff only to the new script being added as part of this PR. We can revisit the other one in a separate PR.